v3/tutorials/05-private-network/index.mdx

```diff
@@ -744,7 +744,8 @@ To add a second validator to the private network:
    --key-type aura
    ```
 
-   Note that this command uses the second participant's secret key and that the `aura` key type is required to enable block production.
+   Replace `<second-participant-secret-seed>` with the secret phrase or secret seed that you generated in [Generate a second  key pair](#generate-second-keypair).
```

Comment:
It should be you can link to this without needing to create an `<a>` tag like you have done above.

If not, that is def a feature request.

Reply:
FIxed the anchor tags.